src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx

```diff
@@ -412,50 +405,49 @@ export const EditVideoForm: React.FC<EditVideoFormProps> = ({
               name="category"
               control={control}
               rules={requiredValidation('Video category')}
-              onFocus={() => handleFieldFocus(categorySelectRef)}
-              render={({ value, onChange }) => (
-                <Select
-                  containerRef={categorySelectRef}
-                  value={value ?? null}
-                  items={categoriesSelectItems}
-                  onChange={(value) => {
-                    onChange(value)
-                    handleFormChange()
-                  }}
-                  error={!!errors.category && !value}
-                  helperText={errors.category?.message}
-                />
-              )}
+              render={({ field: { value, onChange }, fieldState: { invalid } }) => {
+                if (invalid) handleFieldFocus(categorySelectRef)
```

Comment:
This won't work like it used to. Previously we would scroll to the field only when `onFocus` happened, so if the user tried to submit the form that had errors. Here you call the scroll in the `render` function so the scroll will happen every time this field renders which is obviously not what we want to happen. Take a look:

https://user-images.githubusercontent.com/12646744/121869840-bd085280-cd02-11eb-87cd-8961acbe23b4.mov